addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -116,11 +116,20 @@ var RunningTourActionHelper = core.Class.extend({
     },
     _drag_and_drop: function (values, to) {
         var $to;
-        var elementCenter = values.
-        $element.offset();
+        let initialMoveOffset = [1, 0];
```

Comment:
About the whole diff of this file, it looks like you are actually almost not sharing any line between your drag_and_drop and the regular one:
- The first `if (to) {} else {}` is useless in your case as you always have a `to`.
- The `if (parts)` is useless in case of regular.
- You need to add a condition on last lines `if (!fromSelector) {` because useless in your case
- You need to add a condition in the middle `if (fromSelector) {` because useless in original case

What's really shared / left is just the util `calculateCenter()` method.

Based on that, don't you think it would be better to:
1. encapsulate calculateCenter in a method
2. introduced a new method for your new drag & drop and not using the `_drag_and_drop` one. And so leaving that code untouched to keep it simple? I think both methods would then be easier to read.

I ain't no JS expert and @qsm-odoo will probably be doing the final review on the JS part given it is in the `web_tour` module which he knows way better than me, so if you (QSM) do not agree here let BSO know.